placesautocomplete/src/main/java/com/seatgeek/placesautocomplete/PlacesApi.java

```diff
@@ -160,7 +162,8 @@ public PlacesDetailsResponse details(final String placeId) throws IOException {
                 .appendPath(PATH_DETAILS)
                 .appendPath(PATH_JSON)
                 .appendQueryParameter(PARAMETER_KEY, googleApiKey)
-                .appendQueryParameter(PARAMETER_PLACE_ID, placeId);
+                .appendQueryParameter(PARAMETER_PLACE_ID, placeId)
+                .appendQueryParameter(PARAMETER_LANGUAGE, Locale.getDefault().toString());
```

Comment:
@sddamico 
I have update code to use above logic.
Only difference: I have decided do not default languageCode if it is not provided to keep backward compatibility with existing logic.